src/_data/sidenav.yml

```diff
@@ -52,7 +54,7 @@
         - title: Navigation & routing
           permalink: /cookbook/navigation/navigation-basics
         - title: Inspect an app's UI
-          permalink: /development/tools/inspector
+          permalink: /development/ui/inspector # Redirects to /development/tools/inspector
```

Comment:
@galeyang - notice the distinct page path that Firebase redirects to tools.

Reply:
I asked a question in the original issue. #1496